cumulusci/tasks/bulkdata/load.py

```diff
@@ -508,3 +548,128 @@ def _expand_mapping(self):
                         mapping["lookups"][lookup]["after"] = None
 
                     self.after_steps[after][name] = mapping
+
+    def _is_person_account_column_exists(self, mapping):
+        """
+        Returns if "IsPersonAccount" is a column in mapping's table.
+        """
+        return (
+            self.models[mapping.get("table")].__table__.columns.get("IsPersonAccount")
+            is not None
+        )
+
+    def _is_person_accounts_enabled(self, mapping):
```

Comment:
Should we detect that a Person Accounts data store is being loaded into a non-Person Accounts org and stop the operation? I'm not entirely sure what would happen in that situation.

Reply:
It's an interesting idea that the task automatically suppresses loading Person Account `Account` and `Contact` records if person accounts is not enabled in the org.

Reply:
I was thinking more of throwing an exception.

Reply:
Will add check in load in `_run_task` after `_init_db` and before `_expand_mapping` to check if both are true and raise a Bulk Data Exception:
- person accounts is not enabled in the org
- There exists records in the DB where `IsPersonAccount` is true

This means an Exception is raised if person accounts is not enabled in the org but load will try to insert person account records.

Reply:
This is done.